app/controllers/admin/rents_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::RentsController < ApplicationController
+  before_action :login_required
+
+  def main
+    @rents = Rent.ascending.from_date(Time.zone.now.beginning_of_day)
+    @rent_grid = initialize_grid(@rents)
+    @faqs =  Faq.where(answer: '').where(category: 'Bil')
+  end
+
+  def update
+    respond_to do |format|
+      if @rent.update(rent_params)
+        format.html { redirect_to edit_rent_path(@rent), :notice => 'Bokningen uppdaterades.' }
```

Comment:
Line is too long. [95/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.